psl/psl/tests/config/sources.rs

```diff
@@ -758,47 +739,6 @@ fn relation_mode_and_referential_integrity_can_coexist() {
     assert_eq!(config.relation_mode(), Some(RelationMode::ForeignKeys));
 }
 
-#[test]
-fn relation_mode_prisma_has_precedence_over_referential_integrity() {
-    let schema = indoc! {r#"
-        datasource ps {
-          provider = "sqlserver"
-          url = "mysql://root:prisma@localhost:3306/mydb"
-          relationMode = "prisma"
-          referentialIntegrity = "foreignKeys"
-        }
-
-        generator client {
-          provider = "prisma-client-js"
-          previewFeatures = ["referentialIntegrity", "relationMode"]
-        }
-    "#};
-
-    let config = parse_configuration(schema);
-
-    assert_eq!(config.relation_mode(), Some(RelationMode::Prisma));
-}
-
-#[test]
-fn referential_integrity_is_used_if_relation_mode_is_default() {
-    let schema = indoc! {r#"
-        datasource ps {
-          provider = "sqlserver"
-          url = "mysql://root:prisma@localhost:3306/mydb"
-          referentialIntegrity = "prisma"
-        }
-
-        generator client {
-          provider = "prisma-client-js"
-          previewFeatures = ["referentialIntegrity", "relationMode"]
-        }
-    "#};
-
-    let config = parse_configuration(schema);
-
-    assert_eq!(config.relation_mode(), Some(RelationMode::Prisma));
-}
```

Comment:
same as previous comment

Reply:
That one can be removed I think, it's only interesting to keep if we have 2 preview feature flags, and we want to only have 1